lib/kafkajs/JSConsumer.js

```diff
@@ -48,8 +48,7 @@ class JSConsumer extends EventEmitter {
       config.noptions = {};
     }
 
-    const brokers = config.kafkaHost ||
-      (config.noptions["metadata.broker.list"] && config.noptions["metadata.broker.list"].split(","));
+    const brokers = (config.noptions["broker.list"] && config.noptions["broker.list"].split(","));
```

Comment:
the whole noptions block was taken from the native kafka client, which they abbreviated from the native xml kafka options. I would suggest sticking to the same config names (meaning keeping the metadata. prefix) to prevent frustration from people updating to the latest version of our lib